src/components/browse/components/FacetList/FacetTermsList.js

```diff
@@ -370,14 +420,34 @@ const ListOfTerms = React.memo(function ListOfTerms(props){
             );
         }
 
+        // show simple text input for basic search (search within returned values)
+        // or show SAYT control if item search is available
+        let facetSearch = null;
+        if (searchType === 'basic') {
+            facetSearch = (
+                <div style={{ 'padding': '10px', 'fontSize': '0.875rem' }}>
+                    <input className="form-control" autoComplete="off" type="search" placeholder="Search"
+                        name="q" onChange={onBasicTermSearch} key="facet-search-input" />
+                </div>);
+        } else if ((searchType === 'sayt') || (searchType === 'sayt_without_terms')) {
+            const itemType = facet.sayt_item_type && typeof facet.sayt_item_type === 'string' && facet.sayt_item_type !== '' ? facet.sayt_item_type : 'Item';
```

Comment:
Try to use [destructuring](https://itnext.io/how-to-use-object-destructuring-in-modern-javascript-59758ebfb778), will result in shorter and more readable lines of code (albeit sometimes more lines of code)